sceptre/diffing/stack_differ.py

```diff
@@ -169,7 +169,10 @@ def _extract_parameters_from_generated_stack(self, stack: Stack) -> dict:
         for key, value in stack.parameters.items():
             if isinstance(value, list):
                 value = ",".join(item.rstrip("\n") for item in value)
-            formatted_parameters[key] = value.rstrip("\n")
+            if isinstance(value, str):
+                formatted_parameters[key] = value.rstrip("\n")
+            elif value is None:
+                formatted_parameters[key] = ""
```

Comment:
I actually believe what needs to happen here is that we treat `None` as if there is no value at all and don't add it to the `formatted_parameters` dict at all.